anchor/signature_collector/src/lib.rs

```diff
@@ -183,7 +183,7 @@ impl SignatureCollectorManager {
                         {
                             Entry::Occupied(occupied) => occupied,
                             Entry::Vacant(vacant) => vacant.insert_entry(CommitteeSignatures {
-                                signatures: Vec::with_capacity(validators.len()),
+                                signatures: Vec::with_capacity(num_signatures),
```

Comment:
What do you think about renaming `signatures` to `collected_signatures` and `num_signatures` to `num_signatures_to_collect`?